utilities/src/main/java/io/syndesis/qe/bdd/validation/OpenshiftValidationSteps.java

```diff
@@ -36,6 +38,22 @@ public void deployFTPServer() {
         FtpTemplate.deploy();
     }
 
+    @Given("^deploy Kudu rest API")
+    public void deployKuduRestAPI() { KuduRestAPITemplate.deploy(); }
+
+    @Given("^deploy Kudu server")
+    public void deployKuduServer() { KuduTemplate.deploy(); }
```

Comment:
format

Reply:
+ end with $ in the annotation

Reply:
Hmm, the format is the same as other methods in the class, running it through auto format makes some inconsistencies but at least they will be consistent through all the tests ^^

Reply:
idea shows it in single line even if the source file is properly formatted in multiple lines

Reply:
Uhmm... True :) But some stay in long format, no idea why... Anyway it is redone as you wanted it to be :)